htdocs/includes/odtphp/odf.php

```diff
@@ -161,12 +165,17 @@ public function setVars($key, $value, $encode = true, $charset = 'ISO-8859')
 	 */
 	public function convertVarToOdf($value, $encode = true, $charset = 'ISO-8859')
 	{
-		$value = $encode ? htmlspecialchars($value) : $value;
-		$value = ($charset == 'ISO-8859') ? utf8_encode($value) : $value;
+		$value = html_entity_decode($value, ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401);
+
+		// fix breaklines...
+		$value = str_replace("<br>", "<br />", $value);
 		$convertedValue = $value;
 
 		// Check if the value includes html tags
 		if ($this->_hasHtmlTag($value) === true) {
+			// Note: allowing many tags is supported on PHP >7.4 . For older versions, this will strip all HTML tags.
+			$value = strip_tags($value, ['<br>', '<strong>', '<b>', '<i>', '<em>', '<u>', '<s>', '<sub>', '<sup>', '<span>']);
```

Comment:
If instead of using an array parameter
['<br>', '<strong>', '<b>', '<i>', '<em>', '<u>', '<s>', '<sub>', '<sup>', '<span>']
you give the paramater as a string
'<br><strong><b><i><em><u><s><sub><sup><span>'
you will have same result for all php version. Giving array works only with PHP 7.4 but giving a string works on all version. Can you try this ?

Reply:
@eldy thanks for the hint ! I had a wrong understanding of the second parameter of strip_tags before php7.3. I added the changes.